CHANGES.md

```diff
@@ -7,8 +7,9 @@ Release Notes.
 
 #### Project
 
-* Upgrade log4j2 to 2.15.0 for CVE-2021-44228. This CVE only effects on JDK if JDNI is opened in default. Notice, using
-  JVM option `-Dlog4j2.formatMsgNoLookups=true` also avoids CVE if your JRE opened JNDI in default.
+* Upgrade log4j2 to 2.16.0 for CVE-2021-44228 and CVE-2021-45046. This CVE only effects on JDK if JDNI is opened in
+  default. Notice, using JVM option `-Dlog4j2.formatMsgNoLookups=true` or setting
```

Comment:
We are upgrading from 2.9 so the “notice” seems not work at all

Reply:
No, we are not. Check this.

https://github.com/apache/skywalking/blob/v8.9.0/tools/dependencies/known-oap-backend-dependencies.txt#L96-L99